### DISCUSSION TITLE
woowacourse-week-1/

### DISCUSSION THREAD

**Discussion by giscus** [devHudi/hudi.blog#211] [Comments]
# woowacourse-week-1/

https://hudi.blog/woowacourse-week-1/

**Comment by hjlim456** [comment]
안녕하세요 후디님 이번에 우테코 7기 준비중인 취준생입니다.
혹시 프리코스를 잘풀고싶은데 어떻게 준비해야할지 조언좀 해주실수있으실까요?

현재 제 수준은 이전기수 프리코스 문제를 풀어보고 있는데 어디서 부터 구현해야할지 모르는 수준입니다....
오늘 프리코스 후기를 읽으며 설계부터 해야한다는것을 알고 설계를 하고 다른 지원자분들 깃허브에 제출한 코드를 보고 코드분석을 하면서 하나씩 필요한 개념을 찾아보면서 만들고 있는데.. 잘하는방법일까 걱정이 들어서 댓글 남겨봅니다.. 짧게나마 조언 부탁드립니다 ㅠㅠ

(우테코 프리코스 관련 필요한 개념 찾을떄마다 후디님 블로그에서 도움많이 받고있습니다)
